modules/docs/mdx/DOCUMENTATION_GUIDELINES.mdx

```diff
@@ -0,0 +1,238 @@
+<Meta
+  title="Guides/Documentation Guidelines"
+  parameters={{
+    viewMode: 'docs',
+    previewTabs: {
+      canvas: {hidden: true},
+    },
+  }}
+/>
+
+# Documentation Guidelines
+
+## Introduction
+
+### Writing Style
+
+Google provides [helpful resources](https://developers.google.com/tech-writing/overview) for how to
+write technical documentation. We highly recommend you familiarize yourself with their materials
+when authoring documentation for Canvas Kit.
+
+### Conventions
+
+Placeholder content throughout this document is enclosed in brackets as `[placeholder]`. For
+example, if you were writing component documentation for `Tabs`, you would replace:
+
+```tsx
+import {[Component]} from '@workday/canvas-kit-[preview-|labs-]react/[component]';
+```
+
+with:
+
+```tsx
+import {Tabs} from '@workday/canvas-kit-react/tabs';
+```
+
+## Component Documentation
+
+Every component MDX (with a few [exceptions](#component-families)) should follow the structure
+outlined below.
+
+````md
+<!--
+Import components which will be used to render documentation in the MDX file.
+The Specifications import is optional; only include it if the Specifications
+section exists (see below).
+-->
+import {Specifications, SymbolDoc} from '@workday/canvas-kit-docs';
+
+<!-- TODO: Figure out if we need to import the component -->
+<!-- Import component -->
+import {[Component]} from '@workday/canvas-kit-[preview-|labs-]react/[component]';
```

Comment:
As far as I can tell, the imported component is only used in the `<Meta>` Storybook tag a few lines down and even there it doesn't seem to be required (SB renders fine without it). Button.stories.mdx doesn't even include the `component` prop in its [Meta](https://github.com/Workday/canvas-kit/blob/master/modules/react/button/stories/button/Button.stories.mdx#L19) tag -- I'm guessing we left it out because there are multiple Button components, but maybe it's not required in the first place?

Reply:
Never mind, the `component` prop is necessary in order to get the component to render in the Storybook navigation. For example, `<Meta title="Components/Containers/Tabs" />` (without the `component={Tabs}` prop) won't render the `Tabs` section under `Components/Containers`.